client/app/src/time/TimeService.js

```diff
@@ -0,0 +1,60 @@
+(function() {
+  'use strict';
+
+  angular.module('arkclient')
+    .service('timeService', ['$q', '$http', '$interval', TimeService]);
+
+  /**
+   * TimeService
+   * @constructor
+   */
+  function TimeService($q, $http) {
+
+    
+    var timeServerUrl = "http://www.timeapi.org/";
```

Comment:
timeapi.org refused connection

Reply:
Swapped this service with another one. Oddly enough, my development machine can get a response but my other one doesn't? No idea why that's happening.